core/management/generate_data.py

```diff
@@ -0,0 +1,83 @@
+from django.core.management import BaseCommand
```

Comment:
Also the file structure should be: `core/management/commands/generate_data.py` 
And `core`app should be included in `INSTALLED_APPS`Can you fix this as well?